lib/galaxy/auth/providers/ldap_ad.py

```diff
@@ -4,34 +4,37 @@
 @author: Andrew Robinson
 """
 
+import logging
+
 from galaxy.exceptions import ConfigurationError
 from ..providers import AuthProvider
+from galaxy.auth import _get_bool
 
-import logging
 log = logging.getLogger(__name__)
 
 
 def _get_subs(d, k, params):
     if k not in d:
-        raise ConfigurationError("Missing '%s' parameter in Active Directory options" % k)
+        raise ConfigurationError("Missing '%s' parameter in LDAP options" % k)
     return str(d[k]).format(**params)
 
 
-class ActiveDirectory(AuthProvider):
+class LDAPAD(AuthProvider):
+
     """
-    Attempts to authenticate users against an Active Directory server.
+    Attempts to authenticate users against an LDAP server.
 
-    If options include search-fields then it will attempt to search the AD for
-    those fields first.  After that it will bind to the AD with the username
+    If options include search-fields then it will attempt to search LDAP for
+    those fields first.  After that it will bind to LDAP with the username
     (formatted as specified).
     """
-    plugin_type = 'activedirectory'
+    plugin_type = 'ldap_ad'
```

Comment:
``LDAPAD`` is fine for a class/plugin name but I would prefer if it were just ``LDAP``. Can we just make this ``ldap`` and the class ``LDAP``?

Then toward the bottom of the file define

```
class ActiveDirectory(LDAP):
     """ Effectively just an alias for LDAP auth, but may contain active directory specific
     logic in the future. """
     plugin_type = 'activedirectory'

__all__ = ['LDAP', 'ActiveDirectory']
```